Sources/ApolloCodegenLib/Frontend/ApolloCodegenFrontend.swift

```diff
@@ -2,8 +2,16 @@ import Foundation
 import JavaScriptCore
 
 public final class ApolloCodegenFrontend {
-  static let bundle = Bundle(for: ApolloCodegenFrontend.self)
-  private static let libraryURL = bundle.url(forResource: "ApolloCodegenFrontend.bundle", withExtension: "js")!
+  #if SWIFT_PACKAGE
+  private static let bundle = Bundle.module
+  private static let libraryURL = bundle.url(forResource: "ApolloCodegenFrontend.bundle",
```

Comment:
Generally I'm in favor, if there's more than one line in the `#if` statement, of indenting things in `#if/#else` blocks for clarity about what's handled in each scenario. Not a hill I want to die on or anything but I do think it makes things more readable.

Reply:
Yeah, I get that reasoning. The only reason i don't is because Xcode's auto indent is going to change it back to this the next time I hit ctrl+i ... haha. I always try to keep things consistent with the Xcode behavior there, even if I don't love the formatting, because it makes it easier to use auto indentation.

Reply:
fair enough. I wish there was a way to more easily override that ...